test/integration/01-setup/01-funds-bitcoind-lnds.spec.ts

```diff
@@ -17,6 +17,14 @@ jest.mock("src/phone-provider", () => require("test/mocks/phone-provider"))
 
 const defaultWallet = ""
 
+it("mandatory users", async () => {
```

Comment:
This should probably be a `before` block and not an `it` block as there is no expect so nothing is actually tested here.